extensions/amp-analytics/0.1/requests.js

```diff
@@ -87,9 +87,10 @@ export class RequestHandler {
    * @param {?JsonObject} configParams
    * @param {!JsonObject} trigger
    * @param {!./variables.ExpansionOptions} expansionOption
+   * @param {!Object<string, *>} dynamicBindings
```

Comment:
I know none of the other params did it, but would be nice to add a sentence about what this is

Reply:
Done